### PR TITLE
implement V-72095 for stig

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add audit_rules_privileged_commands_mount into  stig profile

- make rule audit_rules_media_export templated

- modify audit_rules_dac_modifications oval template so taht it can be used with vider set of rules (problem with rule names)

#### Rationale:

stig effort

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule audit_rules_media_export:
 Ansible remediation for audit_rules_media_export was deleted.
 OVAL check for audit_rules_media_export was deleted.
 Bash remediation was deleted. No test for it will be selected.
Profile stig on rhel7:
 Rule audit_rules_privileged_commands_mount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_media_export/rule.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_media_export/rule.yml:3]: `rhcos4` product selects this rule too.
This is such a broad rule that I wonder if it makes sense to have `prodtype` in this rule.

**Comment by mildas** [comment]
**Changes identified:**
Rule audit_rules_media_export:
 OVAL check for audit_rules_media_export was deleted.
 Bash remediation was deleted. No test for it will be selected.
 Ansible remediation for audit_rules_media_export was deleted.
Profile stig on rhel7:
 Rule audit_rules_privileged_commands_mount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_media_export/rule.yml (hunk 1 above)
Comment (vojtapolasek) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_media_export/rule.yml]: I am not against, I changed it.

**Pull request merged**